sdk/servicebus/armservicebus/zz_generated_constants.go

```diff
@@ -0,0 +1,411 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package armservicebus
+
+const telemetryInfo = "azsdk-go-armservicebus/<version>"
```

Comment:
After every generation we need to manually change the version number here
```suggestion
const telemetryInfo = "azsdk-go-armservicebus/v0.1.0"
```